modules/gdscript/gdscript_cache.h

```diff
@@ -88,7 +88,7 @@ class GDScriptCache {
 	static Ref<GDScriptParserRef> get_parser(const String &p_path, GDScriptParserRef::Status status, Error &r_error, const String &p_owner = String());
 	static String get_source_code(const String &p_path);
 	static Ref<GDScript> get_shallow_script(const String &p_path, const String &p_owner = String());
-	static Ref<GDScript> get_full_script(const String &p_path, Error &r_error, const String &p_owner = String());
+	static Ref<GDScript> get_full_script(const String &p_path, Error &r_error, const String &p_owner = String(), ResourceFormatLoader::CacheMode p_cache_mode = ResourceFormatLoader::CACHE_MODE_REUSE);
```

Comment:
I assume here that `CACHE_MODE_REUSE` mode was used implicitly before, so this change should be consistent with previous use-cases of `GDScriptCache::get_full_script`